packages/react/src/analytics/integrations/GA4/GA4.ts

```diff
@@ -399,7 +408,18 @@ class GA4 extends integrations.Integration<GA4IntegrationOptions> {
         `context.event.${utils.ANALYTICS_UNIQUE_EVENT_ID}`,
       ) as string;
 
+      const coreVersion = get(
+        data,
+        `context.event.${utils.ANALYTICS_PACKAGE_VERSION}`,
+        '',
+      ) as string;
+
+      const packageVersion = `${PACKAGE_NAME}@${PACKAGE_VERSION};`;
+
       eventProperties[utils.ANALYTICS_UNIQUE_EVENT_ID] = uniqueEventId;
+      eventProperties[
+        'analytics_package_version'
```

Comment:
Why not add 2 separate properties, one for each version?
Also, the properties should start with a `__` to indicate that they are internal.

Reply:
Due last tests, it was informed that properties which begin with _ or __ aren't appear on GA4 Data, so in future we will replace these Analytics_Unique_Event_Id value or update only in GA4 that were this would be an issue.

About 2 separate properties, it's because it was requested to be created a field, named as analytics_package_version to set versions used.

Reply:
Interesting, did not know about those rules for event parameter names but I see they are documented here: https://support.google.com/analytics/answer/13316687#zippy=%2Cweb.

At least then we could use some kind of prefix to make it easier to understand, like `META_analytics_package_version` or something like that.